### DISCUSSION TITLE
How to disable "unselect" in FormBuilderChoiceChip

### DISCUSSION THREAD

**Discussion by wz-naquadah** [danvick/flutter_form_builder#1195] [Q&A] [answered]
I notice in FormBuilderChoiceChip, you can select and unselect options. I'd like to customize the behavior that you can only select one of the provided options, so at least (and at most) one option is selected.

Looking into the source code and searching online doesn't help me find anything that is relevant.

Need some help here.

Thanks,

**Comment by deandreamatias** [comment, marked as answer]
Maybe you can add validators like `min` and `max` and disabled by your own state management when arrive to max select options
